From: Josh Law <objecting@objecting.org>
Subject: lib/uuid: fix typo "reversion" to "revision" in comment
Date: Fri, 6 Mar 2026 16:12:50 +0000

Fix a typo in __uuid_gen_common() where "reversion" (meaning to revert)
was used instead of "revision" when describing the UUID variant field.

Link: https://lkml.kernel.org/r/20260306161250.2811500-1-objecting@objecting.org
Signed-off-by: Josh Law <objecting@objecting.org>
Reviewed-by: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/uuid.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lib/uuid.c~lib-uuid-fix-typo-reversion-to-revision-in-comment
+++ a/lib/uuid.c
@@ -54,7 +54,7 @@ EXPORT_SYMBOL(generate_random_guid);
 static void __uuid_gen_common(__u8 b[16])
 {
 	get_random_bytes(b, 16);
-	/* reversion 0b10 */
+	/* revision 0b10 */
 	b[8] = (b[8] & 0x3F) | 0x80;
 }
 
_
